roles/kubevirt/tasks/provision.yml

```diff
@@ -11,7 +11,7 @@
 
 - name: Create {{ namespace }} namespace
   shell: kubectl create namespace {{ namespace }}
-  when: ns.stdout != "{{ namespace }}"
+  when: ns.stdout != namespace
```

Comment:
I haven't test it but I think this step will be always skipped because ns.stdout equals to `{{ namespace }}`, not `namespace`

Reply:
I think it is ok, I tried that like this:
```yaml
# test.yml
---
- hosts: all
  gather_facts: false
  vars:
    namespace: my-namespace
    stdout: my-namespace
  tasks:
    - name: Print debug message
      debug:
        msg: "namespace: {{ namespace }}, stdout: {{ stdout }}"
    - name: Print only if they are qeual
      debug:
        msg: "stdout == namespace"
      when: stdout == namespace
- hosts: all
  gather_facts: false
  vars:
    namespace: my-namespace
    stdout: other-namespace
  tasks:
    - name: Print debug message
      debug:
        msg: "namespace: {{ namespace }}, stdout: {{ stdout }}"
    - name: Print only if they are qeual
      debug:
        msg: "stdout == namespace"
      when: stdout == namespace
```
```bash
[lbednar@lbednar tmp]$ ansible-playbook -i localhost, --connection local test.yml 

PLAY [all] ************************************************************************************************************************************************************************************

TASK [Print debug message] ********************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "namespace: my-namespace, stdout: my-namespace"
}

TASK [Print only if they are qeual] ***********************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "stdout == namespace"
}

PLAY [all] ************************************************************************************************************************************************************************************

TASK [Print debug message] ********************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "namespace: my-namespace, stdout: other-namespace"
}

TASK [Print only if they are qeual] ***********************************************************************************************************************************************************
skipping: [localhost]

PLAY RECAP ************************************************************************************************************************************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=0   
```

Reply:
it truly works.